src/constants/tokenLists/mainnet.tokenlist.ts

```diff
@@ -121,7 +121,32 @@ export const MAINNET_TOKEN_LIST = {
       name: 'RICE',
       decimals: 18,
       logoURI: 'https://drive.google.com/uc?export=download&id=16PVXI1Da5P27cywWiyqrcV-Q17u1aXsi'
-    }
+    },
+    {
+      chainId: 1,
+      address: '0x0f17bc9a994b87b5225cfb6a2cd4d667adb4f20b',
+      symbol: 'jEUR',
+      name: 'Jarvis Synthetic Euro',
+      decimals: 18,
+      logoURI: 'https://i.imgur.com/ZvP634G'
```

Comment:
Please use the direct link for logoURI
For example, https://i.imgur.com/ZvP634G.png

The same applies to other places